src/main/java/cinema/config/AppConfig.java

```diff
@@ -38,7 +39,7 @@ public LocalSessionFactoryBean getSessionFactory() {
         properties.put("hibernate.show_sql", environment.getProperty("hibernate.show_sql"));
         properties.put("hibernate.hbm2ddl.auto", environment.getProperty("hibernate.hbm2ddl.auto"));
         factoryBean.setHibernateProperties(properties);
-        factoryBean.setPackagesToScan("model");
+        factoryBean.setPackagesToScan(User.class.getPackage().getName());
```

Comment:
краще не прив'язуватись до модельки User, бо вона може змінитись. Тому давай зробимо тут наступне:
```suggestion
        factoryBean.setPackagesToScan("cinema.model");
```

Reply:
fixed